src/plots/cartesian/axes.js

```diff
@@ -3780,12 +3782,24 @@ axes.drawLabels = function(gd, ax, opts) {
                 var pad = !isAligned ? 0 :
                     (ax.tickwidth || 0) + 2 * TEXTPAD;
 
-                var rotate90 = (tickSpacing < maxFontSize * 2.5) || ax.type === 'multicategory' || ax._name === 'realaxis';
+                const adjacent = tickSpacing;
```

Comment:
Currently our build still requires ES5 so it's complaining about `const`, `=>`, and such. I'd love to relax that constraint, and we must be nearly there as we do accept dependencies with modern syntax, but I don't know if you want to dig into that or just convert your new code back to ES5.

Reply:
Let's keep up with ES5 in this PR.

Reply:
I hope I fixed it with my latest commit. I ran eslint with the .eslintrc in the project root on my changed files.